src/main/java/org/ibp/api/rest/program/ProgramUsageResource.java

```diff
@@ -0,0 +1,41 @@
+package org.ibp.api.rest.program;
+
+import io.swagger.annotations.ApiOperation;
+import org.generationcp.middleware.api.program.ProgramDTO;
+import org.ibp.api.java.impl.middleware.security.SecurityService;
+import org.ibp.api.java.program.ProgramService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProgramUsageResource {
+
+	@Autowired
+	private ProgramService programService;
+
+	@Autowired
+	public SecurityService securityService;
+
+	@ApiOperation(value = "Return the last program launched by the user", notes = "Return the last program launched by the user")
+	@RequestMapping(value = "program-usage/last", method = RequestMethod.GET)
```

Comment:
@cuenyad small detail that we have discussed in a daily. To be consistent with the newly created services my-studies and my-lists and given that in both services we are taking the user from the context, I think we should use my-program-usage in both